src/ai/backend/manager/registry.py

```diff
@@ -1246,6 +1246,7 @@ async def start_session(
             'BACKENDAI_CLUSTER_HOSTS':
                 ",".join(binding.kernel.cluster_hostname for binding in kernel_agent_bindings),
             'BACKENDAI_ACCESS_KEY': scheduled_session.access_key,
+            'BACKENDAI_SERVICE_PORTS': ",".join(str(port) for port in scheduled_session.preopen_ports),
```

Comment:
What happens when `scheduled_session.preopen_ports` is `None`?

Reply:
Also, how about add two environment variables:
- `BACKENDAI_SERVICE_PORTS`: Service ports defined in the container image label (`ai.backend.service-ports`).
- `BACKENDAI_PREOPEN_PORTS`: Pre-open ports delivered via the request parameters (`scheduled_session.preopen_ports`).